src/command_modules/azure-cli-container/README.rst

```diff
@@ -0,0 +1,186 @@
+Microsoft Azure CLI 'container' Command Module
+==============================================
+
+Commands to manage Azure container instances
++++++++++++++++++++++++++++++++++++++++++++++
+::
+
+    Group
+        az container: Manage Azure Container Instances.
+
+    Commands:
+        create : Create or update container group.
+        delete : Delete a container group.
+        list   : List container groups by resource group.
+        listall: List all container groups.
+        logs   : Tail the log of a container group.
+        show   : Show the details of a container group.
+
+Commands to create an Azure container group
++++++++++++++++++++++++++++++++++++++++++++++
+::
+
+    Command
+        az container create: Create or update container group.
+
+    Arguments
+        --image             [Required]: The container image name.
+        --name -n           [Required]: The primary resource name.
+        --resource-group -g [Required]: Name of resource group. You can configure the default group
+                                        using `az configure --defaults group=<name>`.
+        --command-line                : The command line to run when the container is started, e.g.
+                                        '/bin/bash -c myscript.sh'.
+        --cpu                         : The required number of CPU cores of the containers.  Default: 1.
+        --environment-variables -e    : A list of environment variable for the container. Space
+                                        separated values in 'key=value' format.
+        --image-registry-login-server : The container image registry login server.
+        --image-registry-password     : The password to log in container image registry server.
+        --image-registry-username     : The username to log in container image registry server.
+        --ip-address                  : The IP address type of the container group.  Allowed values:
```

Comment:
If they anticipate supporting more values in the future, then it would be better to keep it like this.